tests/Unit/Controllers/SettingsControllerTest.php

```diff
@@ -329,7 +418,20 @@ public function setUp(): void
         parent::setUp();
         $this->initDatabase();
 
-        $this->config = new Config(['min_password_length' => 6]);
+        $this->themes = [
+            0 => [
```

Comment:
The explicit numerical naming is not really needed here but ok as its explicitly used as a reference